_includes/portfolio.html

```diff
@@ -0,0 +1,72 @@
+{% if include.portfolio %}
+	{% assign portfolio = include.portfolio %}
+{% else %}
+	{% assign portfolio = page.portfolio.items %}
+{% endif %}
+{% if include.categories %}
+	{% assign categories = include.categories %}
+{% else %}
+	{% assign categories = page.portfolio.categories %}
+{% endif %}
+{% if page.portfolio.button %}
+	{% assign button = page.portfolio.button %}
+{% else %}
+	{% assign button = "Launch Site" %}
+{% endif %}
+
+<div class="portfolio-wrapper showcase">
+	<ul class="filter">
+		<li><a class="active" href="#" data-filter="*">All</a></li>
+		{% for category in categories %}
+			<li><a href="#" data-filter=".{{ category[0] }}">{{ category[1] }}</a></li>
+		{% endfor %}
+	</ul>
+	<ul class="items col4">
+		{% for item in portfolio %}
+		<li class="item {{ item.categories|join: ' ' }}" ata-callback="callPortfolioScripts();" data-detailcontent='
+			<div class="content">
+				{% assign num_images = item.images|size %}
+				<div {% if num_images > 1 %}id="myCarousel" class="carousel slide"{% else %}class="single"{% endif %}>
```

Comment:
It *is* really fast.  We could probably change it somewhere in `scripts.js`.  I don't know if we could make it customizable (or if we'd want to), but I'm sure we'd be able to change it permanently.  I don't like it either, so I'll see if I can track down where it's set.